plugin/main/src/kotlinx/benchmark/gradle/JsEngineExecTasks.kt

```diff
@@ -86,7 +88,15 @@ private fun Project.createNodeJsExec(
     inputFileProperty.set(getExecutableFile(compilation))
     with(nodeArgs) {
         if (compilation.isWasmCompilation) {
-            addWasmArguments()
+            val addGcArgs = project.extensions.findByType(NodeJsRootExtension::class.java)?.let {
+                val nodeVersion = VersionNumber.parse(it.nodeVersion)
+                // Starting from version 22, NodeJS is shipped with V8 versions
```

Comment:
Do we really want to support nodejs version <22? Maybe better just to drop this code as deprecated?

Reply:
Sounds good to me. Those who really need an older version can always supply the experimental-gc flag manually (but we should document that workaround, though).

@qurbonzoda WDYT?

Reply:
I'll use an older version of Node for example. I develop VS Code extensions and my Node version is always synced with the Electron one.

Reply:
@lppedd, I'm curious: does the Kotlin/Wasm work with it?

Reply:
Yes, I can execute K/WASM tests, and I can load WASM binaries inside of my extensions (e.g., inside of a webview).  
See also https://code.visualstudio.com/blogs/2024/05/08/wasm for what VS Code will support going forward.

Hopefully we'll have a VS Code version based on Electron and Node 22+ by the end of the year, but who knows.

Reply:
Given there are use cases, I would prefer to keep the `--experimental-wasm-gc` argument for users on older Kotlin and/or Node.js versions.